web/tests/lynxkite.ts

```diff
@@ -427,7 +427,7 @@ export class BoxEditor extends PopupBase {
   }
 
   isPresent() {
```

Comment:
I think it's better to delete this. You're not even using it.

Please delete `boxExists`/`boxPopupExists` too. Instead use `getBox` and `getBoxEditor` with a `toBeVisible` check. Thanks!